### ISSUE TITLE
AsyncDisplayKit 1.9.8 pod spec is wedged (pre-1.0 pod trunk succeeded, but did not complete the push)

### ISSUE THREAD

**Issue by appleguy** [CocoaPods/CocoaPods#5468]
* [X] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Cocoapods 0.39 resulted in success with "pod trunk push".  Once I realized it did not appear on the website as an available version, I updated to 1.0.1, and then the pod spec lint started failing.  Once that was fixed, now the push is failing with a server side exception.

## What did you expected to happen?

Success pushing 1.9.8

## What happened instead?

** BUILD SUCCEEDED **

 -> AsyncDisplayKit (1.9.8)

opening connection to trunk.cocoapods.org:443...
opened
starting SSL for trunk.cocoapods.org:443...
SSL established
<- "POST /api/v1/pods?allow_warnings=false HTTP/1.1\r\nContent-Type: application/json; charset=utf-8\r\nAccept: application/json; charset=utf-8\r\nUser-Agent: CocoaPods/1.0.1\r\nAuthorization: Token 901a93dc252ed2676187d4a15e99cca4\r\nAccept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3\r\nHost: trunk.cocoapods.org\r\nContent-Length: 1667\r\n\r\n"
<- "{\"name\":\"AsyncDisplayKit\",\"version\":\"1.9.8\",\"license\":{\"type\":\"BSD\"},\"homepage\":\"http://asyncdisplaykit.org\",\"authors\":{\"Scott Goodson\":\"scottgoodson@gmail.com\",\"Ryan Nystrom\":\"rnystrom@fb.com\"},\"summary\":\"Smooth asynchronous user interfaces for iOS apps.\",\"source\":{\"git\":\"https://github.com/facebook/AsyncDisplayKit.git\",\"tag\":\"1.9.8\"},\"documentation_url\":\"http://asyncdisplaykit.org/appledoc/\",\"frameworks\":\"AssetsLibrary\",\"weak_frameworks\":[\"Photos\",\"MapKit\"],\"requires_arc\":true,\"platforms\":{\"ios\":\"7.0\"},\"default_subspecs\":\"PINRemoteImage\",\"social_media_url\":\"https://twitter.com/fbOpenSource\",\"libraries\":\"c++\",\"pod_target_xcconfig\":{\"CLANG_CXX_LANGUAGE_STANDARD\":\"c++11\",\"CLANG_CXX_LIBRARY\":\"libc++\"},\"subspecs\":[{\"name\":\"Core\",\"public_header_files\":[\"AsyncDisplayKit/*.h\",\"AsyncDisplayKit/Details/**/*.h\",\"AsyncDisplayKit/Layout/*.h\",\"Base/*.h\",\"AsyncDisplayKit/TextKit/ASTextNodeTypes.h\",\"AsyncDisplayKit/TextKit/ASTextKitComponents.h\"],\"exclude_files\":[\"AsyncDisplayKit/Private/_AS-objc-internal.h\",\"AsyncDisplayKit/Details/ASDealloc2MainObject.h\",\"AsyncDisplayKit/Details/ASDealloc2MainObject.m\"],\"source_files\":[\"AsyncDisplayKit/**/*.{h,m,mm}\",\"Base/*.{h,m}\",\"AsyncDisplayKit/TextKit/*.h\"],\"dependencies\":{\"AsyncDisplayKit/ASDealloc2MainObject\":[]}},{\"name\":\"ASDealloc2MainObject\",\"requires_arc\":false,\"source_files\":[\"AsyncDisplayKit/Private/_AS-objc-internal.h\",\"AsyncDisplayKit/Details/ASDealloc2MainObject.h\",\"AsyncDisplayKit/Details/ASDealloc2MainObject.m\"]},{\"name\":\"PINRemoteImage\",\"xcconfig\":{\"GCC_PREPROCESSOR_DEFINITIONS\":\"$(inherited) PIN_REMOTE_IMAGE=1\"},\"dependencies\":{\"PINRemoteImage/iOS\":[\">= 3.0.0-beta.2\"],\"AsyncDisplayKit/Core\":[]}}]}\n"
-> "HTTP/1.1 409 Conflict\r\n"
-> "Date: Mon, 06 Jun 2016 21:14:34 GMT\r\n"
-> "Connection: keep-alive\r\n"
-> "Strict-Transport-Security: max-age=31536000\r\n"
-> "Content-Type: application/json\r\n"
-> "Location: https://trunk.cocoapods.org/api/v1/pods/AsyncDisplayKit/versions/1.9.8\r\n"
-> "Content-Length: 73\r\n"
-> "X-Content-Type-Options: nosniff\r\n"
-> "Server: thin 1.6.2 codename Doc Brown\r\n"
-> "Via: 1.1 vegur\r\n"
-> "\r\n"
reading 73 bytes...
-> "{\"error\":\"Unable to accept duplicate entry for: AsyncDisplayKit (1.9.8)\"}"
read 73 bytes
Conn keep-alive
[!] Unable to accept duplicate entry for: AsyncDisplayKit (1.9.8)

### Stack

```
   CocoaPods : 1.0.1
        Ruby : ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin15]
    RubyGems : 2.0.14.1
        Host : Mac OS X 10.11.5 (15F34)
       Xcode : 7.3.1 (7D1014)
         Git : git version 2.7.4 (Apple Git-66)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 60c83329c96804b361a49e38443c54cf67a6e2fb
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.0
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.0.0
cocoapods-try         : 1.0.0
```

### Podfile

```ruby
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '7.0'

target :'AsyncDisplayKitTests' do
  pod 'OCMock', '~> 2.2'
  pod 'FBSnapshotTestCase', '~> 1.8.1'
end
```
## Project that demonstrates the issue

master or git tag '1.9.8'

https://github.com/facebook/AsyncDisplayKit
https://github.com/facebook/AsyncDisplayKit/releases/tag/1.9.8

**Comment by orta** [comment]
I'm confused - https://trunk.cocoapods.org/api/v1/pods/AsyncDisplayKit/versions/1.9.8 was uploaded `2016-06-06 07:54:35 UTC"` 

what do you mean "not available version" also/

**Comment by appleguy** [comment]
https://cocoapods.org/pods/AsyncDisplayKit Check out the version number here...

**Comment by orta** [comment]
This is CocoaPods acting correctly, `1.9.74` is a later SemVer version than `1.9.8` - you can [deprecate or delete](https://guides.cocoapods.org/terminal/commands.html#pod_trunk_delete) `1.9.74` if you want to remove that version?

**Comment by appleguy** [comment]
doh!  Thanks @orta - I expected it would't allow me to push it if that were the case.  Sorry for my confusion :)

**Comment by orta** [comment]
No worries, we'd need to support patch releases for older versions of pods for example - which is why you can push "smaller" SemVer versions than your current

**Comment by appleguy** [comment]
Yeah, we changed the versioning scheme because of Apple rejecting 4-digit version numbers…I think this root issue is now fixed in Cocoapods 1.0 because it no longer passes through the version number into a framework?  If true, we’ll restore versioning sanity after 2.0.

Thanks again!

**Comment by orta** [comment]
Good Q, not sure, think so though

**Comment by segiddins** [comment]
Yup, that is definitely fixed in 1.0
